lib/routes/njxzc/utils/index.ts

```diff
@@ -23,18 +23,33 @@ async function getNoticeList(ctx, url, host, titleSelector, dateSelector, conten
         list.map((item) =>
             cache.tryGet(item.link, async () => {
                 const response = await got(item.link);
-                if (response.redirectUrls.length) {
-                    item.link = response.redirectUrls[0];
+                if (response.redirectUrls && response.redirectUrls.length > 0) {
```

Comment:
`res.redirectUrls` will always be `undefined` since #14922. `'@/utils/got'` is now a partial-got-like wrapper of [ofetch](https://github.com/unjs/ofetch).